app/src/main/res/layout/activity_overlay_transparent.xml

```diff
@@ -47,6 +47,11 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/frameLaout">
 
+        <FrameLayout
+            android:layout_width="48dp"
+            android:layout_height="match_parent">
+        </FrameLayout>
```

Comment:
What is this for?

Reply:
Oh, it can be replaced with a margin or removed entirely. But I thought it would be better to have the seekBar centered and a little bit more space on the left to make it easier to swipe to the minimum.

Reply:
Ah I see, thanks for the explanation. Looks good, so lets keep it ;)